linkerd/config/src/main/scala/io/buoyant/linkerd/config/types/DtabDeserializer.scala

```diff
@@ -11,3 +11,11 @@ class DtabDeserializer extends ConfigDeserializer[Dtab] {
       Dtab.read(_parseString(parser, ctx))
     }
 }
+
+class DtabSerializer extends ConfigSerializer[Dtab] {
```

Comment:
I would prefer splitting the `*Serializer` classes out into their own `.scala` files, to match the Scala Style Guide's [recommendation on file organization](http://docs.scala-lang.org/style/files). Thoughts?